test/tests/hostmask.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+
+const expect = require("chai").expect;
+const Helper = require("../../src/helper");
+
+describe("Hostmask", function() {
```

Comment:
Probably worth colocating the tests in https://github.com/thelounge/thelounge/blob/master/test/src/helperTest.js, and using `describe("#parseHostmask", function() {` (actually should be `".parseHostmask"`).
Thoughts?

Reply:
I really don't mind these function tests being in their own file (a couple of other functions are already separate)

Reply:
Fair enough. At least we should split them per function though.